modules/oe_subscriptions_anonymous/tests/src/Functional/HtmlMailsTest.php

```diff
@@ -279,6 +379,31 @@ protected function assertConfirmMailHtml(EntityInterface $entity, string $mail_b
       $wrapper->text());
   }
 
+  /**
+   * Gets the mail body without the wrapper divs.
+   *
+   * Also asserts the CSS class in the outer wrapper, that derives from the
+   * mail key and module name.
+   *
+   * @param string $mail_id
+   *   Expected mail id/key within oe_subscriptions_anonymous.
+   *
+   * @return string
+   *   Trimmed mail body content without wrapper divs.
+   */
+  protected function getMailBodyWithoutWrapper(string $mail_id): string {
```

Comment:
The goal here was to have more information in failure messages:
If we receive an email with an unexpected class, we see a comparison showing the actual class found.

Reply:
In that case you can add an `assertCount(1, $crawler);`
With targeting class wrapper you expect instead of travelling, and asserting class.
Just an extra check.